templates/auth/apache_auth_publish_api_prefix.conf.epp

```diff
@@ -0,0 +1,27 @@
+<%- |
+  String $api_path,
+  String $auth_type,
+  Stdlib::Absolutepath $auth_file,
+
+  String $prefix,
+  Aptly_profile::PrefixRequires $requires,
+| -%>
+<%
+  $match_prefix = $prefix ? {
+    ''      => ':\.',
+    default => ":${prefix}"
+  }
```

Comment:
Maybe make `$prefix` an `Optional[String]` and parse undef as well?

Reply:
its really more an empty prefix than a undefined prefix imho

Reply:
Sure, was just wondering what would happen in this case if undef gets passed. (though the type will probably fail on that, won't it?)

Reply:
Yes. the type ensures this will never be undef.